extra/3-magic-8-ball.js

```diff
@@ -60,6 +85,34 @@ function shakeBall() {
 */
 function checkAnswer(answer) {
   //Write your code in here
+
+  if (
+    answer == "It is certain." ||
+    answer == "It is decidedly so." ||
+    answer == "Without a doubt." ||
+    answer == "Yes - definitely." ||
+    answer == "You may rely on it."
+  ) {
+    return "very positive";
+  } else if (
+    answer == "As I see it, yes." ||
+    answer == "Most likely." ||
+    answer == "Outlook good." ||
+    answer == "Yes." ||
+    answer == "Signs point to yes."
+  ) {
+    return "positive";
+  } else if (
+    answer == "Reply hazy, try again." ||
+    answer == "Ask again later." ||
+    answer == "Better not tell you now." ||
+    answer == "Cannot predict now." ||
+    answer == "Concentrate and ask again."
+  ) {
+    return "negative";
+  } else {
+    return "very negative";
+  }
```

Comment:
This is fine, but you are using a lot of magic words here. Try to look into what magic words are and how this could be improved.

Hint: 
You can either use a JS object or array index to determine whether it's positive or negative